### PR TITLE
Add screenbuilder type to create command

### PR DESCRIPTION
**Pull request by Fatme**
http://teampulse.telerik.com/view#item/290019

### CODE REVIEW

File: docs/man_pages/project/creation/create.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Usage | Synopsis
 
 General | `$ appbuilder create [<Type>] [<App name>]`
 
-If `<App name>` is specified, new Screen Builder project will be created.
+If only`<App name>` is specified, new Screen Builder project will be created.
```
Comment (teobugslayer) [inline docs/man_pages/project/creation/create.md]: missing space between only and `App

File: docs/man_pages/project/creation/create.md [hunk 2]
```diff
@@ -14,6 +14,7 @@ Creates a project for hybrid, native or mobile website development. You must run
 * `hybrid` - Creates a new project from an **Apache Cordova** template.
 * `native` - Creates a new project from a **NativeScript** template.
 * `website` - Creates a new project from a **Mobile Website** template.
+* `screenbuilder` - Creates  a new **Screen Builder** project.
```
Comment (teobugslayer) [inline docs/man_pages/project/creation/create.md]: two spaces between Creates a

**Comment by rosen-vladimirov** [comment]
new command is added, we need help for it, create-screenbuilder.md file should be created with proper content

File: docs/man_pages/project/creation/create-screenbuilder.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+create
```
Comment (rosen-vladimirov) [inline docs/man_pages/project/creation/create-screenbuilder.md]: title should be create screenbuilder

File: docs/man_pages/project/creation/create-screenbuilder.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+==========
+
+Usage | Synopsis
+------|-------
+
```
Comment (rosen-vladimirov) [inline docs/man_pages/project/creation/create-screenbuilder.md]: remove this blank line

**Comment by rosen-vladimirov** [comment]
:+1:  after passing unit tests

File: docs/man_pages/project/creation/create-screenbuilder.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+Usage | Synopsis
+------|-------
+General | `$ appbuilder create screenbuilder <App name>`
+
+Creates a new Screen Builder project.
+
+### Attributes
+* `<App name>` is the name of the application. The maximum length is 30 characters. You can use only the following characters: A-Z, a-z, 0-9, underscore (_), dot (.), hyphen (-) and space ( ).
+* `<App ID>` must consist of at least three alphanumeric strings, separated by a dot (.). Each string must start with a letter. The application identifier corresponds to the Bundle ID for iOS apps and to the package identifier for Android apps.
```
Comment (teobugslayer) [inline docs/man_pages/project/creation/create-screenbuilder.md]: Do we also support --path and --appid?

Comment (Fatme) [inline docs/man_pages/project/creation/create-screenbuilder.md]: Yes, we support `--path` and `--appid` options

**Comment by Fatme** [comment]
I'll retarget PR against release branch

File: docs/man_pages/project/creation/create-screenbuilder.md (hunk 5 above)
Comment (teobugslayer) [inline docs/man_pages/project/creation/create-screenbuilder.md]: Afaik, they must be documented in the help

**Pull request closed without merging**